test/aoj/grl-6-a.test.cpp

```diff
@@ -0,0 +1,31 @@
+// verification-helper: PROBLEM https://onlinejudge.u-aizu.ac.jp/problems/GRL_6_A
+
+#include <iostream>
+
+#include "src/cpp-template/header/input.hpp"
+#include "src/cpp-template/header/type-alias.hpp"
+#include "src/cpp-template/header/rep.hpp"
+
+#include "src/graph/dinic.hpp"
```

Comment:
すいません　グラフのあたりが順番めちゃくちゃなんですけどこうしようかなと思っております　すいません

```suggestion
#include "src/cpp-template/header/input.hpp"
#include "src/cpp-template/header/type-alias.hpp"
#include "src/cpp-template/header/rep.hpp"
#include "src/graph/dinic.hpp"

#include <iostream>
```